packages/ragbits-core/src/ragbits/core/sources/hf.py

```diff
@@ -32,9 +37,8 @@ def id(self) -> str:
         Returns:
             Unique identifier.
         """
-        return f"{self.protocol}:{self.path}/{self.split}/{self.row}"
+        return f"{self.protocol}:{self.path}/{self.split}" + f"/{self.row}" if self.row is not None else ""
```

Comment:
nit: I don't like having string concat with ternary operator mixed, easy to make errors

```suggestion
return f"{self.protocol}:{self.path}/{self.split}{f'/{self.row}' if self.row is not None else ''}"
```